addons/udes_stock/models/stock_picking_type.py

```diff
@@ -1,4 +1,4 @@
-from odoo import models
+from odoo import models, fields
```

Comment:
Is this needed?

Reply:
Nope, was left in as some fields were added here in an earlier commit in this branch but no longer needed. Have removed. :)